### DISCUSSION TITLE
MDX within Schema

### DISCUSSION THREAD

**Discussion by kenrobbins** [PaloAltoNetworks/docusaurus-openapi-docs#510] [Q&A]
Is there a way to use admonitions or tables inside the description of a schema property?

I noticed that basic things like italics and monospace work but admonitions and tables don't. They work fine outside the schema though, like in the endpoint or response description.

**Comment by sserrata** [comment]
Hi @kenrobbins, your mileage may vary here, since we are using `react-markdown` to render markdown in schema/params items. That said, it should support rendering most (if not all) GitHub-flavored markdown. For things like admonitions, you'd be forced to write/pass them as HTML since they are not supported in GFM. Tables (markdown or HTML) should work.

**Reply by sserrata** [reply]
I should add that it might be possible to override how specific components are rendered. 

See here for an example: https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/packages/docusaurus-theme-openapi-docs/src/theme/SchemaItem/index.js#L62

**Comment by Preston-Landers** [comment]
It seems like I'm able to use admonitions inside schema field descriptions, but only if the field is an expandable one with a fully specified (via items>properties) contents. For regular "scalar" fields, it doesn't seem to render admonitions.

Note: I'm currently using Docusaurus 3.4.0 and docusaurus-plugin-openapi-docs 3.0.1
